src/chrome/content/rules/Uvnc.com.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+	Time out:
+		forum2.uvnc.com
+		mail.uvnc.com
+		support4.uvnc.com
+-->
+<ruleset name="Uvnc.com">
+	<target host="uvnc.com" />
+	<target host="www.uvnc.com" />
+	<target host="forum.uvnc.com" />
```

Comment:
https://travis-ci.org/EFForg/https-everywhere/jobs/586547585#L328
Mismatched: forum, www.mail, sponsor4

Reply:
How to disable them? Just move to comment?